pkgs/development/haskell-modules/tensorflow-packages.nix

```diff
@@ -0,0 +1,310 @@
+{ fetchFromGitHub }:
```

Comment:
I was thinking that these tensorflow packages could probably go in the `non-hackage-packages.nix` file (since that is what they are), but if you also add an update script, it might just be easier to keep them in a separate file.